darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -1270,57 +1270,115 @@ def test_historical_forecast(self, mode):
         if supports_multioutput_natively:
             assert not isinstance(model.model, MultiOutputRegressor)
         else:
             assert isinstance(model.model, MultiOutputRegressor)
 
-    def test_multioutput_validation(self):
-
-        lags = 4
+    model_configs = [(XGBModel, {"tree_method": "exact"})]
+    if lgbm_available:
+        model_configs += [(LightGBMModel, {})]
+    if cb_available:
+        model_configs += [(CatBoostModel, {})]
 
-        models = [
-            XGBModel(
-                lags=lags, output_chunk_length=1, multi_models=True, tree_method="exact"
-            ),
-            XGBModel(
-                lags=lags,
-                output_chunk_length=1,
-                multi_models=False,
-                tree_method="exact",
-            ),
-            XGBModel(
-                lags=lags, output_chunk_length=2, multi_models=True, tree_method="exact"
-            ),
-            XGBModel(
-                lags=lags,
-                output_chunk_length=2,
-                multi_models=False,
-                tree_method="exact",
-            ),
-        ]
-        if lgbm_available:
-            models += [
-                LightGBMModel(lags=lags, output_chunk_length=1, multi_models=True),
-                LightGBMModel(lags=lags, output_chunk_length=1, multi_models=False),
-                LightGBMModel(lags=lags, output_chunk_length=2, multi_models=True),
-                LightGBMModel(lags=lags, output_chunk_length=2, multi_models=False),
-            ]
-        if cb_available:
-            models += [
-                CatBoostModel(lags=lags, output_chunk_length=1, multi_models=True),
-                CatBoostModel(lags=lags, output_chunk_length=1, multi_models=False),
-                CatBoostModel(lags=lags, output_chunk_length=2, multi_models=True),
-                CatBoostModel(lags=lags, output_chunk_length=2, multi_models=False),
-            ]
+    @pytest.mark.parametrize(
+        "config", itertools.product(model_configs, [1, 2], [True, False])
+    )
+    def test_multioutput_validation(self, config):
+        """Check that models not supporting multi-output are properly wrapped when ocl>1"""
+        (model_cls, model_kwargs), ocl, multi_models = config
         train, val = self.sine_univariate1.split_after(0.6)
+        model = model_cls(
+            **model_kwargs, lags=4, output_chunk_length=ocl, multi_models=multi_models
+        )
+        model.fit(series=train, val_series=val)
+        if model.output_chunk_length > 1 and model.multi_models:
+            assert isinstance(model.model, MultiOutputRegressor)
+        else:
+            assert not isinstance(model.model, MultiOutputRegressor)
 
-        for model in models:
-            model.fit(series=train, val_series=val)
-            if model.output_chunk_length > 1 and model.multi_models:
-                assert isinstance(model.model, MultiOutputRegressor)
+    def test_get_multioutput_estimator_multi_models(self):
```

Comment:
can be a bit tricky to understand.

I will add some suggestions which might help for interpretability